hippod/mime_data_db.py

```diff
@@ -141,6 +141,32 @@ def decode_and_write_file(app, sha_sum, data, compress=False):
     return d
 
 
+def already_in_data_db(app, sha):
+    data_db = app['DB_DATA_PATH']
+    data_list = os.listdir(data_db)
+    if sha in data_list:
+        return True
+    return False
+
+
+def higher_mime_type_preference(app, sha, data):
+    # FIXME: hardcoded stuff
+    if data['mime-type'].startswith('x-snippet') or data['mime-type'].startswith('image'):
```

Comment:
why is image required as well? I thought only for x-snippet? More comments make this clear ...